### DISCUSSION TITLE
Could you make it that the `author.name` parameter would be rendered through the markdown parser ?

### DISCUSSION THREAD

**Discussion by slashformotion** [jpanther/congo#17] [Ideas]
You could use this https://gohugo.io/functions/renderstring/#readout

**Comment by jpanther** [comment]
Thanks for the suggestion. I have gone through and added `markdownify` and `emojify` to the author and a few other places that this wasn't already supported. These changes are now on the dev branch and will be included in the next release.

_Ref: 4c13c4d7769309b900c02dcf91869a9e07f8e747_

**Reply by jpanther** [reply]
This is now available in v1.5.0
